### PR TITLE
Update Algorithm Falsy Bouncer

### PR DESCRIPTION
**Pull request by koustuvsinha**
Refer #1094

### CODE REVIEW

File: Algorithm-Falsy-Bouncer.md [hunk 1]
```diff
@@ -65,8 +81,22 @@ function bouncer(arr) {
...
+#### Relevant Links
+
+- [Array.prototype.filter()](JS-Array-Prototype-Filter)
+- [JS Truthy](JS-Truthy)
+- [JS Falsy](JS-Falsy)
+
+
+## :trophy: Credits:
 
 If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @renelis @abhisekp @Rafase282 for your help with Algorithm: Falsy Bouncer`**
```
Comment (atjonathan) [inline Algorithm-Falsy-Bouncer.md]: Add two lines between `on the main chat:` and `Thanks`:
```diff
- If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @renelis @abhisekp @Rafase282 for your help with Algorithm: Falsy Bouncer`**
+ If you found this page useful, you can give thanks by copying and pasting this on the main chat:
+
+ **`Thanks @renelis @abhisekp @Rafase282 for your help with Algorithm: Falsy Bouncer`**
```

**Comment by atjonathan** [comment]
### **LGTM :+1:**

File: Algorithm-Falsy-Bouncer.md [hunk 2]
```diff
@@ -50,6 +64,8 @@ function bouncer(arr) {
```
Comment (atjonathan) [inline Algorithm-Falsy-Bouncer.md]: Add Relevant Links and Code Explanation

File: Algorithm-Falsy-Bouncer.md [hunk 3]
```diff
@@ -37,6 +49,8 @@ function bouncer(arr) {
```
Comment (atjonathan) [inline Algorithm-Falsy-Bouncer.md]: Add Relevant Links and Code Explanation

File: Algorithm-Falsy-Bouncer.md [hunk 4]
```diff
@@ -1,30 +1,42 @@
 # Algorithm Factorialize A Number
 
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
+
 ![](https://i.imgur.com/GluUbAz.jpg)
```
Comment (Rafase282) [inline Algorithm-Falsy-Bouncer.md]: Add the image before the RSA text.

File: Algorithm-Falsy-Bouncer.md (hunk 2 above)
Comment (koustuvsinha) [inline Algorithm-Falsy-Bouncer.md]: @atjonathan the code explanations are consolidated at the end

File: Algorithm-Falsy-Bouncer.md (hunk 3 above)
Comment (atjonathan) [inline Algorithm-Falsy-Bouncer.md]: Add Code Explanation

File: Algorithm-Falsy-Bouncer.md (hunk 2 above)
Comment (atjonathan) [inline Algorithm-Falsy-Bouncer.md]: Add Code Explanation

**Comment by Rafase282** [comment]
The code explanations should be separated. Given they are very similar I would say go in details in one of them and the other not so much details.

**Pull request merged**